app/services/liquid_assigns_service.rb

```diff
@@ -15,4 +18,44 @@ def assigns
 
   def assigns_pro
   end
+
+  # This method uses Liquid::VariableLookup to find all liquid variables from
+  # a given text. We use the list to know which project assign we need.
+  def assigns_from_content
+    return AVAILABLE_PROJECT_ASSIGNS if text.nil?
```

Comment:
should the default be [] rather than a full list?

Reply:
There is a case in Word where `LiquidAssignsService` is invoked globally and there is no text passed (report_processor_job.rb#L25 and the word thorfile Line 51. In those cases, we need to pass all the assigns available.